.github/workflows/test-output.yml

```diff
@@ -0,0 +1,56 @@
+name: Validate JSON
+# adapted from https://github.com/walbo/validate-json.git
+
+on:
+  pull_request:
+  push:
+    branches: 
+      - main
+   
+      
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+
+      - name: install requirements
+        run: pip install -r requirements.txt
+        shell: bash
+
+      - name: set JSON
+        id: step_json
+        run: 
+          python generate_slicer_json.py test_data/terminology_test.csv test.json
+          
+      - uses: actions/checkout@v3
```

Comment:
My understanding is that it generates the test file in a [temp location](https://github.com/Imageomics/slicerMorph_JSON_generator/actions/runs/4927912803/jobs/8805574607?pr=10) and is removed after the test is done. 

In terms of redundancy, I do believe it could be tied into `run-tests.yml` so we don't have to run it multiple times for the same check. I'm not sure the best way to link them (whether by an if statement to generate the JSON validation or to save the artifact of `run-tests.yml` and call it in this workflow). @johnbradley?

Reply:
I think @hlapp is saying there are two places where we are using `actions/checkout@v3` in test-output.yml. You only need to clone the repo once. I think you can remove line 31 since the first step is to checkout the repo
https://github.com/Imageomics/slicerMorph_JSON_generator/blob/61db968eed56a18cfc7f8a6b74e732741f1d0db4/.github/workflows/test-output.yml#L15-L16

Reply:
Thanks! That's a good catch on the extra call floating around.